front_end/css/styles.css

```diff
@@ -462,3 +462,7 @@ div[contenteditable] {
 div[contenteditable] img {
   max-height: 20em;
 }
+
```

Comment:
fixes a bug where traceback errors in student submissions are a mix of monospace and regular font